apm-sniffer/optional-plugins/pom.xml

```diff
@@ -43,6 +43,7 @@
     <modules>
         <module>optional-spring-plugins</module>
         <module>trace-ignore-plugin</module>
+        <module>dubbo-patch</module>
```

Comment:
`dubbo-patch` looks like a strange name. Does this plugin try to resolve dubbo-springmvc conflict? If I am right, please name it `dubbo-springmvc-confict-patch`